prowler/providers/aws/services/elasticache/elasticache_service.py

```diff
@@ -90,6 +97,36 @@ def _describe_replication_groups(self, regional_client):
                     if not self.audit_resources or (
                         is_resource_filtered(replication_arn, self.audit_resources)
                     ):
+                        # Get primary cluster
+                        try:
+                            for node_group in repl_group["NodeGroups"][0]:
+                                primary_node = next(
+                                    (
+                                        node
+                                        for node in node_group["NodeGroupMembers"]
+                                        if node["CurrentRole"] == "primary"
+                                    ),
+                                    None,
+                                )
+                                if primary_node:
+                                    primary_id = primary_node["CacheClusterId"]
+                                    break
+
+                        except Exception as error:
+                            primary_node = repl_group["NodeGroups"][0][
+                                "NodeGroupMembers"
+                            ][0]
+                            primary_id = primary_node["CacheClusterId"]
+                            logger.error(
+                                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+                            )
```

Comment:
Please, get the nodes from `MemberClusters`, and verify within the check if those members have the compliant configuration.